docs/core/track_fitting.md

```diff
@@ -1 +1,61 @@
 # Track Fitting
+The track fitting algorithms estimate the track 
+parameters. It is part of the pattern recognition/track 
+reconstruction/tracking. We can run the 
+track fitting algorithms, 
+after we allocated all hits to single tracks with the 
+help of a track finding algorithm. It is not necessary, 
+that all points of a track are present.
+
+Currently we have implementations for three different 
+fitters:
+* Kalman Filter
+* GSF
+* Global Chi-Square Fitter (GX2F) [wip]
+?? is GSF a least square fit?
```

Comment:
Yes, the GSF is technically a multi-component KF, and thus *theoretically* should give the same results with one component. Due to different material handling this won't be the case however for our implmentations...